ambuda/utils/cheda.py

```diff
@@ -18,56 +18,56 @@ class Token:
 
 
 POS = {
-    "n": "noun",
-    "a": "adjective",
-    "v": "verb",
-    "i": "indeclinable",
-    "va": "participle",
-    "vi": "verbal indeclinable",
+    "n": {"en": "noun", "sa": "subanta"},
+    "a": {"en": "adjective", "sa": "viśeṣaṇa"},
+    "v": {"en": "verb", "sa": "tiṅanta"},
+    "i": {"en": "indeclinable", "sa": "avyaya"},
+    "va": {"en": "participle", "sa": "participle"},
+    "vi": {"en": "verbal indeclinable", "sa": "avyaya"},
 }
 GENDERS = {
-    "m": "masculine",
-    "f": "feminine",
-    "n": "neuter",
-    "": "",
+    "m": {"en": "masculine", "sa": "puṃliṅga"},
+    "f": {"en": "feminine", "sa": "strīliṅga"},
+    "n": {"en": "neuter", "sa": "napuṃsakaliṅga"},
+    "": {"en": "", "sa": ""},
 }
 CASES = {
-    "1": "nominative",
-    "2": "accusative",
-    "3": "instrumental",
-    "4": "dative",
-    "5": "ablative",
-    "6": "genitive",
-    "7": "locative",
-    "8": "vocative",
-    "c": "compounded",
+    "1": {"en": "nominative", "sa": "prathamā vibhakti"},
```

Comment:
Just prathamā should be sufficient -- see https://ashtadhyayi.com/shabda/01.005 e.g.